apps/nrm/models/submission.js

```diff
@@ -0,0 +1,183 @@
+  response['Customer.County'] = data['pv-contact-details-county'];
+  response['Customer.Postcode'] = data['pv-contact-details-postcode'];
+  response['Customer.ContactMethod'] = data['pv-contact-details'];
+  response['Customer.Alias'] = data['pv-name-nickname'];
+  if (data['pv-dob']) {
+    response['Customer.DOBKnown'] = 'Yes';
+    response['Customer.DOB'] = data['pv-dob'];
+  }
+  // response['Customer.AgeGroup'];
+  const genderMap = {
+    'female': 'Female',
+    'male': 'Male',
+    'unknown': 'Other'
+  };
+  response['Customer.iGender'] = genderMap[data['pv-gender']];
+  response['Customer.Email'] = data['pv-contact-details-email-input'];
+  response['Customer.Mobile'] = data['pv-phone-number-yes'];
+  response['Customer.Mobile'] = data['pv-phone-number-yes'];
+  // response['Customer.ContactTime'];
+  response['Customer.HaveChildren'] = data['does-pv-have-children'] === 'yes' ? 'Yes' : 'No';
+  response['Customer.NumberOfChildren'] = data['does-pv-have-children-yes-amount'];
+  response['Customer.Nationality'] = data['pv-nationality'];
+  response['Customer.DualNationality'] = data['pv-nationality-second'];
+  // response['Customer.Country'];
+  // response['Customer.ReferenceNumbers'] = data['pv-ho-reference-type'];
+  response['Customer.InterpreterNeeded'] = data['pv-interpreter-requirements'] === 'yes' ? 'Yes' : 'No';
+  response['Customer.InterpreterLanguage'] = data['pv-interpreter-requirements-language'];
+  response['Customer.HelpWithCommunication'] = data['pv-other-help-with-communication'] === 'yes' ? 'Yes' : 'No';
+  response['Customer.CommunicationAid'] = data['pv-other-help-with-communication-aid'];
+  // response['Customer.PassportNumber'];
+
+  response.AdultOrChild = data['pv-under-age'] === 'yes' ? 'Child' : 'Adult';
+  response.AdultOrChildDuringExploitation = data['pv-under-age-at-time-of-exploitation'] === 'yes' ? 'Child' : 'Adult';
+  response.VictimAccount = data['what-happened'];
+  response.ExploitationLocationPresented = data['where-exploitation-happened'].toUpperCase();
+
+  response.City1 = data['where-exploitation-happened-uk-city-1'];
+  response.City2 = data['where-exploitation-happened-uk-city-2'];
+  response.City3 = data['where-exploitation-happened-uk-city-3'];
+  response.City4 = data['where-exploitation-happened-uk-city-4'];
+  response.City5 = data['where-exploitation-happened-uk-city-5'];
+  response.City6 = data['where-exploitation-happened-uk-city-6'];
+  response.City7 = data['where-exploitation-happened-uk-city-7'];
+  response.City8 = data['where-exploitation-happened-uk-city-8'];
+  response.City9 = data['where-exploitation-happened-uk-city-9'];
+  response.City10 = data['where-exploitation-happened-uk-city-10'];
+
+  // response.Region1;
+  // response.Region2;
+  // response.Region3;
+  // response.Region4;
+  // response.Region5;
+  // response.Region6;
+  // response.Region7;
+  // response.Region8;
+  // response.Region9;
+  // response.Region10;
```

Comment:
comment?

Reply:
yes because it's in the API but we don't know what to do with it yet